acquia/acquia_cms
# ACMS-1705: regenerate site studio package with dependencies

**Motivation**

Fixes #[ACMS-1705](https://backlog.acquia.com/browse/ACMS-1705)

**Proposed changes**

This change help export drupal standard config and site studio configuration in separate directory and its provides drush post command that import/export site studio package on config import/export.

**Alternatives considered**
Manually do the configuration and run drush command separately.

**Testing steps**
export your site studio keys
`./vendor/bin/acms site:install --yes`
`./vendor/bin/drush sitestudio:package:export`
include `$settings['site_studio_sync'] = '../config/cohesion/sync';` in settings.php
verify all packages with dependencies are getting exported in site studio sync directory.

**Merge requirements**
